### PR TITLE
Implement translation for tint text

### PR DESCRIPTION
**Pull request by satrun77**

### CODE REVIEW

**Comment by jonom** [comment]
This looks neat :) I'm a bit concerned about the extra http requests, especially since this PR doesn't include any extra lang files besides english, so feels a little redundant. Do you want to add an extra language file while you're at it?

How much extra JS / http requests gets loaded with this change? If it's a lot, maybe we could use the PHP/yml i18n implementation instead, and load the definitions on the field as data elements?

Would you mind making the definitions a little more semantic, like:
```
'TextTargetLength.LengthTooShort': "Keep going!",
'TextTargetLength.LengthTooLong': "Too long",
'TextTargetLength.LengthAdequate': "Okay",
'TextTargetLength.LengthIdeal': "Great!"
```

Thanks for this

**Comment by satrun77** [comment]
With Javascript translation, 2 files are loaded for current locale and a fallback. I have updated the PR to use data attributes.

Sorry can't help with adding more translations. But this is a start for others to contribute.

File: javascript/text-target-length.js [hunk 1]
```diff
@@ -13,20 +13,20 @@
...
                     remarkClass = 'under';
                     if (charCount === 0) remark = '';
 				} else if (charCount > max) {
-    				remark = 'Too long'
+    				remark = field.data('hintLengthToolong');
                     remarkClass = 'over';
 				}
 				countEl.attr('class', remarkClass + ' target-length-count');
-				countEl.html('Length target: <b>' + targetFulfilled + '%</b> <i>' + remark + '</i>');
+				countEl.html(ss.i18n.sprintf(field.data('hintLengthTarget'), targetFulfilled, remark));
```
Comment (jonom) [inline javascript/text-target-length.js]: Thanks @satrun77, think this line just needs updating to remove the ss.i18n dependancy and then we should be good to go!

**Comment by satrun77** [comment]
I have update the PR to use replace method instead of SilverStripe sprintf.

**Comment by jonom** [comment]
Thanks!

**Pull request merged**